build/website_cron_testing/push_image.sh

```diff
@@ -19,6 +19,11 @@ set -e
 
 gcloud config set project datcom-ci
 
+cp ../../server/integration_tests/standalone/nodejs_query.py .
```

Comment:
this is a bit hacky, worth to find a better solution later.

Reply:
this script is now removed